### PR TITLE
Update protocol flag docs

### PR DESCRIPTION
**Pull request by frdomovic**
# Update protocol flag docs

## Summary:
Add description and explanations for "--protocol" flag in the process of node initialization
Screenshot 2024-11-05 at 15 40 57

### CODE REVIEW

File: docs/03-getting-started/01-setup.mdx [hunk 1]
```diff
@@ -61,6 +61,23 @@ cargo run -p merod -- --node-name node1 --home data run
 
 Node is now initialized and ready for use.
 
+> **_NOTE:_** A `--protocol` flag is also available to specify your preferred
```
Comment (MatejVukosav) [inline docs/03-getting-started/01-setup.mdx:64]:
```suggestion
> **_NOTE:_** `--protocol` flag defines 
```

File: docs/03-getting-started/01-setup.mdx [hunk 2]
```diff
@@ -61,6 +61,23 @@ cargo run -p merod -- --node-name node1 --home data run
...
+> protocol for the context. Currently, we support Starknet and NEAR protocol. By
+> default, the protocol is set to NEAR protocol, but you can set it explicitly
+> to Starknet by using the following command:
+>
+> ```bash title="Terminal"
+> cargo run -p merod -- --node-name node1 --home data init --server-port 2428 --swarm-port 2528 --protocol starknet
+> ```
+>
+> If you want to use NEAR protocol, you don’t need to specify the --protocol
```
Comment (MatejVukosav) [inline docs/03-getting-started/01-setup.mdx:73]:
```suggestion
> If you want to use NEAR protocol, you don’t need to specify the --protocol
```

redundant, already mentioned

File: docs/03-getting-started/01-setup.mdx [hunk 3]
```diff
@@ -61,6 +61,23 @@ cargo run -p merod -- --node-name node1 --home data run
...
+> flag, as it’s the default protocol used by your node.
+>
+> You can also view these settings in the config.toml file located within the
+> node folder in the storage directory.
```
Comment (MatejVukosav) [inline docs/03-getting-started/01-setup.mdx:77]:
```suggestion
> node folder.
```

File: docs/03-getting-started/01-setup.mdx (hunk 1 above)
Comment (frdomovic) [reply docs/03-getting-started/01-setup.mdx]: updated

File: docs/03-getting-started/01-setup.mdx (hunk 2 above)
Comment (frdomovic) [reply docs/03-getting-started/01-setup.mdx]: updated

File: docs/03-getting-started/01-setup.mdx (hunk 3 above)
Comment (frdomovic) [reply docs/03-getting-started/01-setup.mdx]: updated

**Review by miraclx — APPROVED** [review]

**Review by alenmestrov — APPROVED** [review]

**Pull request merged**